tests/helpers.py

```diff
@@ -56,8 +56,10 @@ def maybe_async_cm(obj: ContextManager[T] | AsyncContextManager[T]) -> AsyncCont
 def get_exception_group() -> type[BaseException]:
     """Get the exception group class with version compatibility."""
     try:
-        return cast("type[BaseException]", globals()["__builtins__"]["ExceptionGroup"])
+        eg = globals()["__builtins__"]["ExceptionGroup"]
     except KeyError:
```

Comment:
```suggestion
        eg = ExceptionGroup
    except NameError:
```

?

Or even `if sys.version_info >= (3, 11):`?
Current way is cryptic.

Reply:
You are right, it is weird to do it this way.

It originates from the pyright PR as for some reason they report
```
/litestar/tests/helpers.py:59:14 - error: "ExceptionGroup" is unbound (reportUnboundVariable)
```

I'll ignore the pyright error.

Reply:
Oh, actually, it is a runtime error:

```
Traceback:
tests/helpers.py:59: in get_exception_group
    eg = ExceptionGroup  # pyright: ignore[reportUnboundVariable]
E   UnboundLocalError: cannot access local variable 'ExceptionGroup' where it is not associated with a value
```